addons/sale_expense/tests/test_reinvoice.py

```diff
@@ -82,7 +121,7 @@ def test_expenses_reinvoice(self):
                     'name': 'expense_1 invoicing=order, expense=sales_price',
                     'date': '2016-01-01',
                     'product_id': self.company_data['service_order_sales_price'].id,
-                    'quantity': 1,
+                    'total_amount': 100.34,
```

Comment:
This test is imbitable.
I have no idea what is being tested and how to see that it follows what I expect from it.
And you're making it worse to understand.